assets/javascripts/running.js

```diff
@@ -939,6 +946,12 @@ var messageToStatusVariable = [
     {
         msg: 'current_api_function',
         statusVar: 'currentApiFunction',
+        action: function (value, data) {
+            developerMode.currentApiFunctionArgs = '';
+            if ((value == 'assert_screen' || value == 'check_screen') && data['check_screen']) {
```

Comment:
Use `===` instead of `==`.
Use `data.check_screen` instead of `[]` braces (also for the line below).

Reply:
I've made the code style changes and added a simple test for currentApiFunctionArgs handling. However, there should be one more subtest for array of tags, e.g.:

    {"data":{"check_screen":{"check":0,"mustmatch":["generic-desktop","emergency-shell","emergency-mode"],"timeout":400},"current_api_function":"assert_screen"},"type":"info","what":"cmdsrvmsg"}

Unfortunately, I'm not sure how to compare arrays correctly. The is() function will just compare array references which doesn't work.

Reply:
You could use `is_deeply` instead (which works on references).

Reply:
Done.